eng/packaging.targets

```diff
@@ -54,29 +76,55 @@
              Targets="Build" />
   </Target>
 
-  <Target Name="AddRuntimeSpecificAssemblies"
-          DependsOnTargets="BuiltProjectOutputGroup;SatelliteDllsProjectOutputGroup;$(TargetsForTfmSpecificBuildOutput)"
-          Condition="'$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+  <Target Name="AddDocumentationFileToPackage">
+    <PropertyGroup>
+      <!-- Search for the documentation file in the intellisense package and otherwise pick up the generated one. -->
+      <_documentationFileToPackage Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">$(XmlDocFileRoot)1033\$(TargetName).xml</_documentationFileToPackage>
+      <_documentationFileToPackage Condition="'$(_documentationFileToPackage)' == '' and Exists('$(DocumentationFile)')">$(DocumentationFile)</_documentationFileToPackage>
```

Comment:
we should add a property to always use the generated file. i think @safern was working on us being able to use the generated doc files.

Reply:
Yes, there are some projects where the intellisense package has a matching xml, but that we already ported the docs to triple slash, but we haven't added support for those to "override" the one from the intellisense package.

Reply:
I just mimic the existing behavior as there isn't such a switch today. Your comment is valid but unrelated to this PR IMO.